runner.go

```diff
@@ -20,7 +20,6 @@ import (
 	"github.com/hashicorp/consul-template/config"
 	dep "github.com/hashicorp/consul-template/dependency"
 	"github.com/hashicorp/consul-template/watch"
-	shellwords "github.com/mattn/go-shellwords"
```

Comment:
It looks like this might be the only import of `go-shellwords`. Suggestion to run a `go mod tidy` to clean this up

Reply:
I think I forgot to run a go mod tidy after the general update. I'm going to fix that and rebase.

Reply:
Yep. that removed it. Thanks!